patches/minecraft/net/minecraft/client/renderer/entity/RenderItem.java.patch

```diff
@@ -9,7 +9,61 @@
          this.func_175041_b();
      }
  
-@@ -226,7 +226,7 @@
+@@ -153,7 +153,7 @@
+ 
+                 if (p_180454_1_.func_77962_s())
+                 {
+-                    this.func_180451_a(p_180454_2_);
++                    this.func_180451_a(p_180454_2_, p_180454_1_);
+                 }
+             }
+ 
+@@ -161,33 +161,18 @@
+         }
+     }
+ 
+-    private void func_180451_a(IBakedModel p_180451_1_)
++    private void func_180451_a(IBakedModel p_180451_1_, ItemStack stack)
+     {
+-        GlStateManager.func_179132_a(false);
```

Comment:
Ehh I've really got little reason to dig more haha my knowledge of rendering pipelines is next to nil :P I was just pushing this for more modding power with the item/ armor rendering. IE being able to add enchantment shaders and such as opposed to boosting vanilla performance